src/api/v1beta1/oneagent_types.go

```diff
@@ -29,6 +29,11 @@ type OneAgentSpec struct {
 }
 
 type CloudNativeFullStackSpec struct {
+	// Optional: the Dynatrace installer container image
```

Comment:
the description isn't quite accurate:
- this doesn't default to anything, but needs to be set by the user to a private registry

Reply:
ah yes thank you for reminding me.